app/services/affiliate-slots-targeting.json

```diff
@@ -41,5 +41,12 @@
     "wikiId": [
       673
     ]
+  },
+  {
+    "campaign": "ddb",
+    "category": "ddb",
+    "wikiId": [
+      51
```

Comment:
What wiki is this?

Reply:
Just realized this was for filtering. Will remove because will be driven by Taxonomy ™️